### PR TITLE
Added Event Tracing

### PR DESCRIPTION
**Pull request by lfourie**
Signed-off-by: lfourie <louis.fourie@huawei.com>

### CODE REVIEW

**Comment by duglin** [comment]
LGTM

File: use-cases.md [hunk 1]
```diff
@@ -78,6 +78,11 @@ A common event format would allow easier reasoning about the data being
 transited and allow for better introspection of the data.
 
 ### Event Tracing
+An event sent from a source may result in a sequence of additional events
```
Comment (SeanFeldman) [inline use-cases.md]: Nit: for consistency sake, should there be a break line between header and paragraph?

Comment (lfourie) [reply use-cases.md]: Will fix

**Comment by ac360** [comment]
LGTM.  Nice one, @lfourie!

**Comment by duglin** [comment]
@lfourie if you update with the small nit we should be able to merge tomorrow

**Comment by duglin** [comment]
@lfourie thanks for the quick fix

**Comment by duglin** [comment]
@lfourie sorry - there's a conflict - can you rebase?

**Comment by duglin** [comment]
Approved on the 3/22/2018 call - will merge after the rebase.

**Comment by lfourie** [comment]
@duglin what is best procedure for the rebase?

**Comment by duglin** [comment]
@lfourie   In my env what I do is:
- update master to the latest
- git checkout BRANCH_NAME
- git rebase -i origin/master
- it should complain about conflicts, edit the spec.md to deal with the conflicts (look for "<<")
- git add spec.md
- git rebase --continue
- git push origin BRANCH_NAME

**Comment by duglin** [comment]
oops s/spec.md/use-cases.md/

Although in this case you may need to edit and move the file since I think the conflict is with Rachel's moving of it.

**Comment by duglin** [comment]
@lfourie I think your rebase grabbed other people's commits by mistake.
Would you like me to give it a shot?

**Comment by lfourie** [comment]
@duglin yes

**Comment by duglin** [comment]
@lfourie I think I got it - take a look and make sure I didn't mess it up

**Comment by lfourie** [comment]
@duglin look good

**Comment by duglin** [comment]
cool - merging since we agreed to the text yesterday

**Pull request merged**